### PR TITLE
Split python into 2 packages

### PR DESCRIPTION
**Pull request by renefritze**

### CODE REVIEW

**Comment by renefritze** [comment]
The clang jobs actually just fail somewhat randomly w/o explanation. I guess OOM. I'm about to disable them for now

**Comment by renefritze** [comment]
> The clang jobs actually just fail somewhat randomly w/o explanation. I guess OOM. I'm about to disable them for now

Instead I updated clang from 13 to 15 in the base image. Which seems to work, just much slower (2h 28m 41s vs 1h 51m 56s)

**Comment by renefritze** [comment]
> > The clang jobs actually just fail somewhat randomly w/o explanation. I guess OOM. I'm about to disable them for now
> 
> Instead I updated clang from 13 to 15 in the base image. Which seems to work, just much slower (2h 28m 41s vs 1h 51m 56s)

Spoke too soon. Only worked for cpp. Disabling now anyways.

File: python/CMakeLists.txt [hunk 1]
```diff
@@ -1,21 +1,23 @@
 # ~~~
 # This file is part of the dune-gdt project:
-#   https://github.com/dune-community/dune-gdt
+# https://github.com/dune-community/dune-gdt
```
Comment (ftschindler) [inline python/CMakeLists.txt:3]: Just saw this by chance. Looks like somethings off with our license header script. Ideas, @renefritze?

Comment (renefritze) [reply python/CMakeLists.txt:3]: I'm afraid that's my VS Code that thinks it's fixing markdown headings or some such.

Comment (renefritze) [reply python/CMakeLists.txt:3]: pylicense will fix that again though

**Comment by renefritze** [comment]
[Publishing](https://github.com/dune-gdt/dune-merged/actions/runs/3289088837/jobs/5422330243)
works for [xt](https://test.pypi.org/project/dune.xt/2022.3.3.post2.dev19/) and [gdt](https://test.pypi.org/project/dune.gdt/2022.3.3.post2.dev19/)

**Comment by ftschindler** [comment]
very nice

**Pull request merged**